server/wfprev-api/src/test/java/ca/bc/gov/nrs/wfprev/entities/CulturalPrescribedFireReportEntityTest.java

```diff
@@ -115,7 +115,7 @@ void testAllFields() {
         assertThat(entity.getSecondaryObjectiveTypeDescription()).isEqualTo("Secondary Obj");
         assertThat(entity.getEndorsementTimestamp()).isEqualTo(now);
         assertThat(entity.getApprovedTimestamp()).isEqualTo(now);
-        assertThat(entity.getOutsideWuiInd()).isEqualTo("Y");
+        assertThat(entity.getOutsideWuiInd()).isEqualTo(true);
```

Comment:
We might want to display "Yes" instead of true or false in the excel form. 
Can we convert OutsideWuiInd to "Yes"/ "No" when popolating to the report?